obj/mainDashBoard.js

```diff
@@ -0,0 +1,17 @@
+const functions = {
+    waitForPage: function () {
+      return this.waitForElementVisible('@providerName');
+    },
+    orderFirstitem: function () {
+        this.click('@firstMenuItem');
+        this.click('@confimOrderBtn');
+    },
+  }
+  module.exports = {
+    commands: [functions],
+    elements: {
+        providerName: '.headline.red--text',
+        firstMenuItem: '.container.fluid > div:nth-child(1) > div:nth-child(3) > span:nth-child(1) > div:nth-child(1)',
```

Comment:
This locator type very much depending on page layout and not recomended. ".dish-card" would be enough to find cards and then with help of xpath you could select first of them